hotness/anitya.py

```diff
@@ -215,10 +215,18 @@ def add_new_project(self, name, homepage):
         )
 
         # Try to guess at what backend to prefill...
-        for target, backend in backends.items():
-            if target in homepage:
+
+        # Start with package prefix first...
+        for prefix, backend in prefixes.items():
+            if name.startswith(prefix) and backend != None:
                 data['backend'] = backend
-                break
+
+        # And try homepage if backend was not detected based on prefix
+        if 'backend' not in data:
+            for target, backend in backends.items():
+                if target in homepage:
+                    data['backend'] = backend
+                    break
```

Comment:
I might recommend trying the homepage before trying the backend though I could see an argument for doing it this way. As a packager, if I set the home page to the GitHub page, I might want GitHub tags to be the thing that is used. Also, not all ```python-``` packages that are on GitHub would necessarily be on PyPI (for example, I know this PR is just for Ruby for now).